subsys/net/lib/fota_download/src/fota_download.c

```diff
@@ -29,6 +30,8 @@ LOG_MODULE_REGISTER(fota_download, CONFIG_FOTA_DOWNLOAD_LOG_LEVEL);
 static fota_download_callback_t callback;
 static const char *dl_host;
 static const char *dl_file;
+static uint32_t dl_host_sha256;
+static uint32_t dl_file_sha256;
```

Comment:
```suggestion
static uint32_t dl_host_hash;
static uint32_t dl_file_hash;
```

You have misleading names here (and below). We use 32bit hash, not SHA256.

Reply:
Fixed